### ISSUE TITLE
Missing Stuff

### ISSUE THREAD

**Issue by justusschock** [justusschock/PyTorch-Network-Wrapper#1]
Missing:

- [ ] Tensor Transformations
- [ ] General Purpose Data Loader
- [ ] Tensorboard integration
- [ ] Learning rate scheduler
- [ ] Variety of Predefined networks (U-Net, GAN, etc)
